### ISSUE TITLE
Project.getPath is deprecated.

### ISSUE THREAD

**Issue by GregDuhamel** [s-shin/linter-perl#7]
Use ::getPaths instead
```
Project.getPath (C:\Users\Gregory\AppData\Local\atom\app-0.190.0\resources\app\src\project.js:585:12)
LinterPerl.setupCommand (C:\Users\Gregory\.atom\packages\linter-perl\lib\linter-perl.coffee:24:32)
```

**Comment by s-shin** [comment]
Thank you for reporting! I'll fix it.

**Comment by s-shin** [comment]
Fixed @f84ee5eff208d4e746baf14d3b85cdff1334df6e
